### PR TITLE
Add Alibaba cloud aliyun-cli

### PR DESCRIPTION
**Pull request by nikita-bykov**
Added Alibaba cloud aliyun-cli for
- Ubuntu 16.04
- Ubuntu 18.04
- Windows 2016
- Windows 2019

**choco** was used for installation on Windows  and the installation package was downloaded on Ubuntu.

Issue: #237

### CODE REVIEW

File: images/linux/scripts/installers/aliyun-cli.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+################################################################################
+##  File:  aliyun-cli.sh
+##  Desc:  Installs Alibaba Cloud CLI
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install Alibaba Cloud CLI
+URL=$(curl -s https://github.com/aliyun/aliyun-cli | grep -m1 ">Linux<" | cut -d "\"" -f 2)
```
Comment (alepauly) [inline images/linux/scripts/installers/aliyun-cli.sh:11]: is this resilient enough? would it make sense to go to the Releases instead to download from there?

File: images/linux/scripts/installers/aliyun-cli.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+################################################################################
+##  File:  aliyun-cli.sh
+##  Desc:  Installs Alibaba Cloud CLI
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# Install Alibaba Cloud CLI
+page_URL=$(curl -s https://github.com/aliyun/aliyun-cli/releases/latest | cut -d "\"" -f 2)
```
Comment (alepauly) [inline images/linux/scripts/installers/aliyun-cli.sh:11]: I think depending on the html to find things will break eventually. We should be using the Releases API which will even give you a download URL. see: https://api.github.com/repos/aliyun/aliyun-cli/releases/latest

Comment (nikita-bykov) [reply images/linux/scripts/installers/aliyun-cli.sh]: Thanks! Fixed it using Releases API.

File: images/win/Windows2016-Azure.json [hunk 3]
```diff
@@ -731,6 +737,18 @@
                 "{{ template_dir }}/scripts/Installers/Validate-Bazel.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "scripts":[
+                "{{ template_dir }}/scripts/Installers/Validate-AliyunCli.ps1"
+            ]
+        },
+        {
```
Comment (alepauly) [inline images/win/Windows2016-Azure.json:746]: looks like the merge brought the SQL Express lines back, watch out! 🍕

**Pull request merged**